### PR TITLE
INT-B-20477

### PR DESCRIPTION
**Pull request by loganwc**
## Summary

PPM Status Enum had a typo

### CODE REVIEW

File: pkg/models/ppm_shipment.go [hunk 1]
```diff
@@ -73,7 +73,7 @@ type PPMShipmentStatus string
 
 const (
 	// PPMShipmentStatusCancelled captures enum value "DRAFT"
```
Comment (pambecker) [inline pkg/models/ppm_shipment.go:75]: can you correct this comment too while you are changing it.  copy and paste error with "DRAFT"

Comment (loganwc) [reply pkg/models/ppm_shipment.go]: good catch, updated

Comment (pambecker) [reply pkg/models/ppm_shipment.go]: I'm not seeing the update, did you push it up?

File: pkg/models/ppm_shipment.go [hunk 2]
```diff
@@ -72,8 +72,8 @@ type PPMSITEstimatedCostInfo struct {
 type PPMShipmentStatus string
 
 const (
-	// PPMShipmentStatusCancelled captures enum value "DRAFT"
-	PPMShipmentStatusCancelled PPMShipmentStatus = "CANCELLED"
+	// PPMShipmentStatusCanceled captures enum value "DRAFT"
+	PPMShipmentStatusCancelled PPMShipmentStatus = "CANCELED"
```
Comment (pambecker) [inline pkg/models/ppm_shipment.go:76]: Actually, I'm just now seeing, this variable name still has 2 'll's, can you update that also, to match the enum value.  You will also have to update it in move_canceler.go, but should be a simple change.

**Pull request merged**